[TASK] Revise chapter about XML sitemaps (#2150)
TYPO3-Documentation/TYPO3CMS-Reference-CoreApi: Documentation/ApiOverview/Seo/XmlSitemap.rst

..  include:: /Includes.rst.txt
..  index:: XML sitemap
..  _xmlsitemap:

===========
XML sitemap
===========

It is possible to generate XML sitemaps for SEO purposes without using 3rd party
plugins. When this feature is enabled, a sitemap index file is created with one
or more sitemaps in it. By default, there will be one sitemap that contains all

> pages of the current site and language. Per each site and language, you can
> render a different sitemap.

Reviewer: ```suggestion
pages of the current site and language.  You can render different sitemaps
for each site and language.
```

.. note::
   The XML sitemap is provided by the optional system extension
   EXT:seo. You can find information about how to install and use it in the
   :doc:`EXT:seo manual <ext_seo:Index>`.

.. contents:: Table of Contents
   :depth: 1
   :local:

Installation
============

> The XML sitemap is not available by default. You can easily activate it by

Reviewer: What do you consider default? In the TYPO3/DEFAULT package of the composer helper seo is already contained and activated: https://get.typo3.org/misc/composer/helper

Author: I considered typo3/minimal: https://packagist.org/packages/typo3/minimal

Author: Maybe: "The seo system extension provides XML sitemaps. If it is not available on your installation, you can easily ..."

Reviewer: Better, but avoid easily, this frustrates people who will have trouble installing

> available in your installation, it can be easily activated by installing it and
> including the static TypoScript template `XML Sitemap (seo)`.

Reviewer: ```suggestion
available in your installation, require it as described here: :ref:`Installation, EXT:seo <ext_seo:installation>`
Then include the static TypoScript template `XML Sitemap (seo)`.
```
